sormas-api/src/main/resources/strings.properties

```diff
@@ -521,15 +524,15 @@ infoUploadDocumentTemplate=Select a ".docx"-Document Template you would like to
-infoCompletenessMerge = You can use this measurement to get an idea about which case you should use as the one that remains active when merging. However, please make sure that you still have a detailed look at both cases to avoid mistakes.
-infoMergeIgnoreRegion = Selecting this option excludes the region as a criteria for detecting duplicates. Merging and picking are deactivated here because the probability of a real duplicate beyond region boundaries is very low. If a real duplicate is detected, this case can be deleted manually via the case directory.
+infoMergingExplanation = This view is designed to assist you in detecting and merging duplicate entities. entity are always displayed as pairs, the entity with the earlier creation date being on top.<br/><br/>You can use the creation date filter to only view potential duplicates that were created in a specific time period. The creation date is automatically generated when a entity is entered into SORMAS and can not be changed by users. Therefore, once you have reviewed all duplicates from a specific time period in the past, you will never have to do it again as there will be no new entities created during this time frame.<br/><br/>Please note that most likely not all displayed entities are actual duplicates. There is the potential for false positives, so <b>please thoroughly review the entities before merging them!</b> In addition to the information provided in the table, you can also click on the IDs of the entities to open them in a new tab. Keep in mind that it is much worse to falsely merge unique entities together than to have some duplicates in the system.
+infoMergingMergeDescription = Choose this option only if you are <b>sure that the two cases/contacts are the same!</b> You should click on this button next to the entity with the most recent or more complete information. Click on the IDs of the entities to open them in new tabs in order to view all their data. When the entity you chose is missing information that is present in the other entity, it will be updated. Also, contacts, samples, tasks and case management information will be transfered. Afterwards, the case you did not choose will be deleted.
+infoMergingPickDescription = Clicking on this button will simply delete the entity that was not picked. However, no information will be copied to the entity that remains active. All contacts or cases, samples and tasks will be lost as well. Choose this option if you are sure that both entities are the same, but the other entity has not been updated with any information that needs to be transfered to the one that remains active.
```

Comment:
Same problem here. Cases are never lost anyway, and contacts aren't lost when merging contacts because contacts don't have contacts.

Reply:
infoMergingPickDescription = Clicking on this button will simply delete the entity that was not picked. However, no information will be copied to the entity that remains active. All samples and tasks will be lost as well. Choose this option if you are sure that both entities are the same, but the other entity has not been updated with any information that needs to be transfered to the one that remains active.       Is this OK for you ?

Reply:
Yep!